templates/commands/render.go

```diff
@@ -47,28 +48,29 @@ func (r *Render) Desc() string {
 
 func (r *Render) Help() string {
 	return `
-Usage: {{ COMMAND }} [options]
-
-  The {{ COMMAND }} command renders the given template into the given dest directory.
-`
+Usage: {{ COMMAND }} [options] <source>
+
+The {{ COMMAND }} command renders the given template.
+
+The "<source>" is the location of the template to be instantiated. Many forms
+are accepted:
+
+  - "helloworld@v1" means "github.com/abcxyz/helloworld repo at revision v1;
+    this is for a template owned by abcxyz.
+  - "myorg/helloworld@v1" means github.com/myorg/helloworld repo at revision
+    v1; this is for a template not owned by abcxyz but still on GitHub.
+  - "mygithost.com/mygitrepo/helloworld@v1" is for a template in a remote git
+    repo but not owned by abcxyz and not on GitHub. 
+  - "mylocaltarball.tgz" is for a template not in git but present on the local
+    filesystem. 
+  - "http://example.com/myremotetarball.tgz" os for a non-Git template in a
+    remote tarball.`
 }
 
 func (r *Render) Flags() *cli.FlagSet {
 	set := cli.NewFlagSet()
 
 	f := set.NewSection("RENDER OPTIONS")
-	f.StringVar(&cli.StringVar{
-		Name:    "source",
-		Aliases: []string{"s"},
-		Example: "helloworld@v1",
-		Target:  &r.flagSource,
-		Usage: `Required. The location of the template to be instantiated. Many forms are accepted. ` +
-			`"helloworld@v1" means "github.com/abcxyz/helloworld repo at revision v1; this is for a template owned by abcxyz. ` +
-			`"myorg/helloworld@v1" means github.com/myorg/helloworld repo at revision v1; this is for a template not owned by abcxyz but still on GitHub. ` +
-			`"mygithost.com/mygitrepo/helloworld@v1" is for a template in a remote git repo but not owned by abcxyz and not on GitHub. ` +
-			`"mylocaltarball.tgz" is for a template not in git but present on the local filesystem. ` +
-			`"http://example.com/myremotetarball.tgz" os for a non-Git template in a remote tarball.`,
-	})
```

Comment:
If you upgrade to the latest CLI package and add a `PredictArgs()` function, you can get tab completion: https://github.com/abcxyz/pkg/blob/bd3d55380569e64a7c5a3acf2ff1bd81f502f8dd/cli/command_completions_doc_test.go#L47-L51